### ISSUE TITLE
RNTester-Windows-Android

### ISSUE THREAD

**Issue by TendiF** [facebook/react-native#19654]
## Environment
Environment:
  OS: Windows 10
  Node: 10.4.0
  Yarn: Not Found
  npm: 6.1.0
  Watchman: Not Found
  Xcode: N/A
  Android Studio: Not Found

Packages: (wanted => installed)
  react: 16.3.1 => 16.3.1
  react-native: 0.55.4 => 0.55.4

## Description
After Following instruction in Running on Android Section [here](https://github.com/facebook/react-native/tree/master/RNTester)

`./gradlew :RNTester:android:app:installDebug` work well for installing app, and the app icon shows up as well in my emulator and devices too.

![s80611-21095483](https://user-images.githubusercontent.com/19442589/41236877-63430112-6dbc-11e8-9123-aa6cf1a27388.jpg)

but i got some error while run `./scripts/packager.sh` whiches run same as `node ./local-cli/cli.js start` i think.

![error](https://user-images.githubusercontent.com/19442589/41237196-295697d8-6dbd-11e8-9a26-baafa04bcc73.png)

error text output

<b>C:\Users\icreativelabs\Documents\react-native\local-cli\cliEntry.js:28
import type {CommandT} from './commands';
       ^^^^
SyntaxError: Unexpected identifier
    at new Script (vm.js:74:7)
    at createScript (vm.js:246:10)
    at Object.runInThisContext (vm.js:298:10)
    at Module._compile (internal/modules/cjs/loader.js:670:28)
    at Module._compile (C:\Users\icreativelabs\Documents\react-native\node_modules\pirates\lib\index.js:91:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Object.newLoader [as .js] (C:\Users\icreativelabs\Documents\react-native\node_modules\pirates\lib\index.js:96:7)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
PS C:\Users\icreativelabs\Documents\react-native></b>

and i dont have an idea what happen, because i init new react native app 
 and run npm start or `node node_modules/react-native/local-cli/cli.js start`
here result.

![work](https://user-images.githubusercontent.com/19442589/41237571-0b114e16-6dbe-11e8-86c2-7e0838f9f09e.png)

**Comment by hramos** [comment]
It's preferable to copy and paste the actual text output here into the issue. You can use a \<detail\>\</detail\> box if it's too much text. It will allow others to stumble upon this issue using search, and it's also way more readable than a screenshot.

**Comment by TendiF** [comment]
I copy and paste the error output, can see under error screenshot. Hope that can be help

**Comment by gengjiawen** [comment]
The script is written for unix-like system. I am working on a pr to make it work on windows. You can check it here [RNTester-windows](https://github.com/gengjiawen/react-native/tree/feature/RNTester_start).  But I run into a issue in metro https://github.com/facebook/metro/issues/181. You can try release build, but I run into some weird gradle error on windows https://github.com/facebook/react-native/pull/19758.

**Comment by lahed** [comment]
Same error react-native init

**Comment by Dimous** [comment]
Any progress on this?
_Windows users can relate to airbnb._

**Comment by gengjiawen** [comment]
I created a pr for this issue https://github.com/facebook/react-native/pull/19974, but now it blocked by an metro issue.

**Comment by tbkunboy** [comment]
run $ react-native init --version 0.55.0

**Comment by gengjiawen** [comment]
The pr has been complete, after it got merged you can take a try.

**Comment by pvdz** [comment]
I'm pretty sure the original reported problem has been fixed by updating Babel past the point of introducing the problem. If the issue is still open now (Metro 42.2, RN 0.57) please open a new ticket.

**Comment by gengjiawen** [comment]
@pvdz Nope, you can't run a bash file on windows. It's not just the type thing. The main purpose here is to debug RNTester on windows.
